### DISCUSSION TITLE
AWS Spring Cloud Copyright

### DISCUSSION THREAD

**Discussion by danihg6** [awspring/spring-cloud-aws#255] [Q&A]
Hi all,
I m a solutions architect and 1 of our teams wants to use Spring cloud in AWS environment, but our internal open source office needs some information for compliance and I can't find where the copyright or author/ authors for spring cloud is indicated. Something like this for example: "Copyright (C) 2010 ..." or author xxx

Kind regards

**Comment by maciejwalkowiak** [comment]
Isn't license (https://github.com/awspring/spring-cloud-aws/blob/2.4.x/LICENSE.txt) enough? Also, each file has a copyright header.

**Reply by danihg6** [reply]
Thanks for your response Maciej, but in most cases, for exemple springdoc openapi ui or springdoc openapi data rest, I found this kind of information on file: 
"* Copyright 2019-2020 the original author or authors."  and when I search in the same file no author is mentioned.

**Reply by maciejwalkowiak** [reply]
I think this is a standard. License should be enough. Spring Boot follows same patterns: license file + this header in each file.
